ev3dev/display.py

```diff
@@ -267,3 +277,77 @@ def update(self):
             self.mmap[:] = self._img_to_rgb565_bytes()
         else:
             raise Exception("Not supported")
+
+    def image_filename(self, filename, clear_screen=True, x1=0, y1=0, x2=None, y2=None):
+
+        if clear_screen:
+            self.clear()
+
+        filename_im = Image.open(filename)
+
+        if x2 is not None and y2 is not None:
+            return self._img.paste(filename_im, (x1, y1, x2, y2))
+        else:
+            return self._img.paste(filename_im, (x1, y1))
+
+    def shapes_line(self, clear_screen=True, x1=0, y1=0, x2=0, y2=0, line_color='black', width=1):
```

Comment:
Why not `draw_line`, `draw_circle`, etc.?

Reply:
I just went went the names the @dlech had in the image posted in https://github.com/rhempel/ev3dev-lang-python/issues/356  I'm not sure if those are official names of the EV3-G equivalents or not though.  I agree though `draw_line`, etc sounds better.

Reply:
This is where it comes from:

![image](https://user-images.githubusercontent.com/3310349/31523789-8543762c-af6a-11e7-850e-64c197f2f3b3.png)

I don't think calling them `draw_line`/`draw_circle`/etc. will be confusing. It might even work to just call it `line`/`circle`...